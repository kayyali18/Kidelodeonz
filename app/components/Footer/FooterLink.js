import { Link } from 'react-router-dom'
import styled from 'styled-components'

export default styled(Link)`
  display: inline-flex;
  /* padding: 0.25em 2em; */
  height: 90%;
  align-items: center;
  justify-content: center;
  text-align: center;
  text-decoration: none;
  border-radius: 0px;
  -webkit-font-smoothing: antialiased;
  -webkit-touch-callout: none;
  user-select: none;
  cursor: pointer;
  outline: 0;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  font-weight: bold;
  font-size: calc(10px + 6 * ((100vw - 320px) / 680));
  border: none;
  color: #fff;
  width: calc(0.3 * 100vw);

  &:hover {
    background: #fff;
    color: #ff700b;
  }
`
