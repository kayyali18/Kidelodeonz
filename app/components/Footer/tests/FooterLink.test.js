import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import FooterLink from '../FooterLink'

describe('<FooterLink />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = shallow(<FooterLink />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
