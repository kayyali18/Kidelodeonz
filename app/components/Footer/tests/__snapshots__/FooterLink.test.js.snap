// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterLink /> should match the snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "FooterLink-sc-1gbpsk1-0",
        "isStatic": true,
        "rules": Array [
          "display:inline-flex;height:60%;align-items:center;justify-content:center;text-align:center;text-decoration:none;border-radius:0px;-webkit-font-smoothing:antialiased;-webkit-touch-callout:none;user-select:none;cursor:pointer;outline:0;font-family:'Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:bold;font-size:calc(10px + 6 * ((100vw - 320px) / 680));border:none;color:#fff;width:calc(0.3 * 100vw);&:hover{background:#fff;color:#ff700b;}",
        ],
      },
      "displayName": "FooterLink",
      "render": [Function],
      "styledComponentId": "FooterLink-sc-1gbpsk1-0",
      "target": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    }
  }
  forwardedRef={null}
/>
`;
